Src/StackifyLib/Utils/HelperFunctions.cs

```diff
@@ -304,4 +313,40 @@ public static string CleanPartialUrl(string url)
             return sbNewUrl.ToString();
         }
     }
+
+    public class ToStringConverter : JsonConverter
+    {
+        private string _propName = "";
+        private Type _type;
+
+        public ToStringConverter(string propName, Type t)
+        {
+            _propName = propName;
+            _type = t;
+        }
+
+        public override bool CanConvert(Type objectType)
+        {
+            return _type.IsAssignableFrom(objectType);
+        }
+
+        public override bool CanRead => false;
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            if (value != null)
```

Comment:
Why not just do this (Remove `_propName`):

```c#
            if (value == null)
                writer.WriteNull();
            else
                writer.WriteValue(value.ToString());
```

Reply:
For now, I want to keep the propname for filtering and consistency.